### PR TITLE
Add Django 2.x support

### PR DESCRIPTION
**Pull request by Aiky30**
The package does not allow Django 2.2 to be installed. 

This fix reuses the changes made for Django CMS 3.6+: https://github.com/divio/aldryn-django-cms/pull/113

### CODE REVIEW

**Comment by FinalAngel** [comment]
👍

**Review by FinalAngel — APPROVED** [review]

File: aldryn_config.py [hunk 1]
```diff
@@ -88,24 +88,19 @@ def to_settings(self, data, settings):
...
-            'cms.middleware.page.CurrentPageMiddleware',
-            'cms.middleware.toolbar.ToolbarMiddleware',
-            'cms.middleware.language.LanguageCookieMiddleware',
+        settings['TEMPLATES'][0]['OPTIONS']['context_processors'].extend([
+            'sekizai.context_processors.sekizai',
+            'cms.context_processors.cms_settings',
         ])
-        settings['MIDDLEWARE_CLASSES'].insert(0, 'cms.middleware.utils.ApphookReloadMiddleware',)
+
+        if settings.get('MIDDLEWARE', None):
```
Comment (viktor-yunenko) [inline aldryn_config.py]: @Aiky30 , would that mean that if MIDDLEWARE doesn't have a default value then cms classes could be not added? If yes, and eg this logic relies on another package to add a default MIDDLEWARE value, perhaps a logger.error could be added when it doesn't happen?

Comment (Aiky30) [reply aldryn_config.py]: @FinalAngel I took this snippet from the following commit that appears to ensure that the MIDDLEWARE setting has been initialised with a value before trying to add values to it, do you know which scenario this caters for?

https://github.com/divio/aldryn-django-cms/pull/113/commits/ef4cbe5feffe444767fade79e0cb9b80269e7db6

Comment (Aiky30) [reply aldryn_config.py]: The commit that I refer to is taken from the following PR: https://github.com/divio/aldryn-django-cms/pull/113

Comment (Aiky30) [reply aldryn_config.py]: @victor-yunenko You may be on to something here as the latest version supports the existing MIDDLEWARE_CLASSES and MIDDLEWARE: 

https://github.com/divio/aldryn-django-cms/blob/support/3.7.x/aldryn_config.py#L94

Going by the logic above this snippet should always be ran without the condition.

@FinalAngel Please confirm and I will remove the condition altogether. I believe that as MIDDLEWARE_CLASSES is now deprecated we don't need to keep the existing definition alive and just remove the condition. https://docs.djangoproject.com/en/2.2/topics/http/middleware/#upgrading-pre-django-1-10-style-middleware

Please advise

Comment (viktor-yunenko) [reply aldryn_config.py]: Hm, I agree, to me it looks like it was a django 1.11 compatibility clause

Comment (FinalAngel) [reply aldryn_config.py]: I think that was still added as compatibility for 3.6 and is still relevant for 3.7 but not for 4.0 anymore.

Comment (Aiky30) [reply aldryn_config.py]: @victor-yunenko @FinalAngel Please see that I have updated the code to remove that logic and the cloud project has been redeployed to verify that the build still continues to fail on the same section.

Comment (viktor-yunenko) [reply aldryn_config.py]: Looks good to me 👍

**Review by viktor-yunenko — APPROVED** [review]

**Pull request merged**